meta/processor/src/main/kotlin/net/skyscanner/backpack/ksp/BackpackSymbolProcessor.kt

```diff
@@ -40,28 +42,30 @@ class BackpackSymbolProcessor(
 
     override fun process(resolver: Resolver): List<KSAnnotated> {
         if (invoked) return emptyList()
-        // if we run the ksp for test sources we will generate empty stories list
-        if (!resolver.getAllFiles().any { "main" in it.filePath }) return emptyList()
 
         val filer = XProcessingEnv.create(environment.options, resolver, environment.codeGenerator, environment.logger).filer
```

Comment:
The logic of the processor has been changed a bit.

First, we find all the component annotations (this step remains the same)
Second, we find all the custom story annotations declared, and parse its params
Finally, we find all the stories annotated with custom annotations and generate the code